packages/integration-sdk-runtime/src/fileSystem.ts

```diff
@@ -127,7 +127,15 @@ export async function symlink({ sourcePath, destinationPath }: SymlinkInput) {
   const fullDestinationPath = path.resolve(directory, destinationPath);
 
   await ensurePathCanBeWrittenTo(fullDestinationPath);
-  await fs.symlink(fullSourcePath, fullDestinationPath, 'junction');
+  // On Windows, we need to perform hardlinks for files
+  if (
+    (await fs.lstat(fullSourcePath)).isFile() &&
+    process.platform === 'win32'
```

Comment:
Why is `win32` included in this condition?

Reply:
I included the `process.platform === 'win32'` conditional so that we could continue to use symlinks for files when developing in non-Windows platforms.

Reply:
Interesting that possible values for [process.platform](https://nodejs.org/api/process.html#processplatform) doesn't include `win64` 🤷

Apparently, [`win32` includes 64 bit platforms](https://stackoverflow.com/questions/8683895/how-do-i-determine-the-current-operating-system-with-node-js). TIL!

Thanks for answering my question 👍

Reply:
:)  Yep!  I tested on a 64 bit Windows build and realized the same thing.  Doesn't feel great seeing 32 bit seemingly explicitly called out, but that's how process.platform is currently designed.